.github/workflows/cross-version-tests.yml

```diff
@@ -60,8 +60,26 @@ jobs:
         run: |
           conda install python=3.6
           python --version
+      - name: Get cache key
+        env:
+          INSTALL_COMMAND: ${{ matrix.install }}
+        id: get-cache-key
+        run: |
+          date=$(date -u "+%Y%m%d")
+          hash=$(echo -n "$INSTALL_COMMAND" | sha256sum)
+          echo "::set-output name=key::$ImageOS-$ImageVersion-wheels-$date-$hash"
```

Comment:
`ImageOS` and `ImageVersion` are pre-defined in the github actions virtual environment.